server/test/src/main/java/io/deephaven/server/test/FlightMessageRoundTripTest.java

```diff
@@ -482,7 +481,7 @@ public void testLoginHeaderAnonymous() {
         final String ANONYMOUS = "Anonymous";
 
         closeClient();
-        scriptSession.setVariable("test", TableTools.emptyTable(10).update("I=i"));
+        scriptSession.getVariableProvider().setVariable("test", TableTools.emptyTable(10).update("I=i"));
```

Comment:
Should this match the others?

Reply:
Oddly, unlike the others, this passes as written. Will investigate.